### DISCUSSION TITLE
Endre på Stepper

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1895] [Komponenter]
🔗 [Link til Stepper i Aksel](https://aksel.nav.no/komponenter/core/stepper)

Vi ønsker å vurdere hva som kan forbedre brukeropplevelsen av Stepper for utviklere og designere. Har du innspill til noe som vil gjøre denne komponenten bedre og mer brukervennlig for deg og brukerne av deres løsninger?

**Comment by macbruker** [comment]
Jeg foreslår å dempe det visuelle uttrykket til Stepper. Det tror jeg vil gjøre at annet innhold får sterkere fokus, som i mange tilfeller er ønskelig.

Grepene jeg har sett på: 

* Fjerne border rundt tallene
* Dempe streken mellom stegene
* Redusere bruken av semibold



![image](https://user-images.githubusercontent.com/1032194/229118966-20f56163-1168-447c-8b1a-5cbdbed74ed6.png)
